utils/frame/remote-externalities/src/lib.rs

```diff
@@ -157,10 +157,14 @@ impl Transport {
 			} else {
 				uri.clone()
 			};
-			let http_client = HttpClientBuilder::default().build(uri).map_err(|e| {
-				log::error!(target: LOG_TARGET, "error: {:?}", e);
-				"failed to build http client"
-			})?;
+			let http_client = HttpClientBuilder::default()
+				.max_request_body_size(u32::MAX)
+				.request_timeout(std::time::Duration::from_secs(60 * 5))
```

Comment:
Is this needed? I would rather have it fail after 1 min if the node is down, and not wait another 4.

Reply:
You can remove if you want but if you do big batch request is could take longer :P

We bumped into this in the staking miner against public RPC nodes IIRC

Reply:
Yea but we have the re-try feature :sunglasses: